### DISCUSSION TITLE
Sokobond

### DISCUSSION THREAD

**Discussion by Leilei332** [Gamuxorg/bbs#284] [投递和请求游戏]
游戏名称：Sokobond
游戏名称(英文)：Sokobond
下载地址 :（如果有的话）：https://rutracker.org/forum/viewtopic.php?t=4853470
magnet:?xt=urn:btih:69AFEE8871C81991CF31C9A3779EC0EAE0BD773F&tr=http%3A%2F%2Fbt2.t-ru.org%2Fann%3Fmagnet&dn=Sokobond%20%5Bx86%5D%20%5BENG%5D%20%5BAir%5D
官网：https://www.sokobond.com
Steam：http://store.steampowered.com/app/290260/
网盘：https://www.jianguoyun.com/p/DRBbUo8QnNWwCRiAr4ME
>（只有tar.gz版本，流量用完了，里面自带Adobe Air的deb版本）

**Comment by shouhuanxiaoji** [comment]
收到

**Comment by shouhuanxiaoji** [comment]
正在处理

**Comment by shouhuanxiaoji** [comment]
已添加，但由于adobe air问题，是x86 32位
